torch/_tensor.py

```diff
@@ -60,7 +60,7 @@ def __deepcopy__(self, memo):
             # does accurate alias tracking; however, the code below
             # doesn't work because of
             # https://github.com/pytorch/pytorch/issues/47442
-            if self.is_sparse or self.device.type in ['xla', 'mlc', 'meta']:
+            if self.is_sparse or self.device.type in ['xla', 'mlc', 'ort', 'meta']:
```

Comment:
ort doesn't have the concept of storage?

Reply:
Similar to XLA, we have not exposed the storage behind the `OrtValue` that backs the `ORTTensorImpl` through `c10::TensorImpl::storage()`, etc.